contracts/LifCrowdsale.sol

```diff
@@ -22,7 +23,21 @@ contract LifCrowdsale is Ownable, Pausable {
 
   // address where funds are collected
   address public foundationWallet;
-  address public marketMaker;
+
+  // minimun amount of wei to be raised in order to succed, it starts in USD
+  uint256 public maxCapPresaleUSD = 1000000;
+
+  // minimun amount of wei to be raised in order to succed, it starts in USD
+  uint256 public minCapUSD = 5000000;
+
+  // maximun balance that the foundation can have, it starts in USD
+  uint256 public foundationBalanceCapUSD = 10000000;
```

Comment:
What about `minMarketMakerCapUSD`, so it's more consistent with the next variable?

Reply:
I can rename it to `maxFoundationCapUSD` if you want, but I think its more important that we cant have more than 10M usd rather than the minimun balance for the MM is10M.

Reply:
Ok, that's a good reason, let's use `maxFoundationCapUSD` as you suggest